core/shared/src/main/scala/zio/Fiber.scala

```diff
@@ -197,8 +197,8 @@ trait Fiber[+E, +A] { self =>
     UIO.effectTotal {
       val p: concurrent.Promise[A] = scala.concurrent.Promise[A]()
 
-      def failure(cause: Exit.Cause[E]): UIO[p.type] = UIO(p.failure(cause.squashWith(f)))
-      def success(value: A): UIO[p.type]             = UIO(p.success(value))
+      def failure(cause: zio.Cause[E]): UIO[p.type] = UIO(p.failure(cause.squashWith(f)))
```

Comment:
Like @iravid mentioned, it would be nice to shorten these to `Cause` instead of `zio.Cause`, both here and elsewhere.